src/colorizer/Dataset.ts

```diff
@@ -395,7 +394,7 @@ export default class Dataset {
       this.loadFeature(data)
     );
 
-    const result = await Promise.all([
```

Comment:
Promise.all fails if any of the promises fail, while Promise.allSettled waits for all promises to succeed OR fail and then return the resulting statuses.

Reply:
Not shown in the edits, but this allows line 404, `this.loadFrame(0)` to be fine if it throws an error.